server.js

```diff
@@ -77,6 +77,11 @@ io.on('connection', function(socket) {
     io.emit('chatMessage', data);
   });
 
+  //Emits alert message only to sender
+  socket.on('sendAlert', function(data) {
+    socket.emit('chatMessage', data);
```

Comment:
Should this be io.sockets.connected[socket.id].emit('chatMessage', data); instead? so that it only sends back to the client who sends it